content/well-architected-framework/docs/docs/secure-systems/infrastructure/manage-network-ingress-egress.mdx

```diff
@@ -0,0 +1,168 @@
+---
+page_title: Secure network traffic with ingress and egress
```

Comment:
Is this title missing a word? Or should it be something along the lines of "Secure ingress and egress network traffic with <thing/concept>"